store/postgres/inter_project_job_spec_repository.go

```diff
@@ -0,0 +1,49 @@
+package postgres
+
+import (
+	"context"
+
+	"gorm.io/gorm"
+
+	"github.com/odpf/optimus/models"
+)
+
+type interProjectJobSpecRepository struct {
+	db      *gorm.DB
+	adapter *JobSpecAdapter
+}
+
+func (repo interProjectJobSpecRepository) GetJobByName(ctx context.Context, jobName string) ([]models.JobSpec, error) {
+	var jobs []Job
+	var specs []models.JobSpec
+	if err := repo.db.WithContext(ctx).Preload("Namespace").Preload("Project").Where("name = ?", jobName).Find(&jobs).Error; err != nil {
+		return []models.JobSpec{}, err
+	}
+	for _, job := range jobs {
+		adapt, err := repo.adapter.ToSpec(job)
+		if err != nil {
+			return specs, err
+		}
+		specs = append(specs, adapt)
+	}
+	return specs, nil
+}
+
+func (repo interProjectJobSpecRepository) GetJobByResourceDestination(ctx context.Context, resourceDestination string) (models.JobSpec, error) {
+	var job Job
+	if err := repo.db.WithContext(ctx).Preload("Namespace").Preload("Project").Where("destination = ?", resourceDestination).First(&job).Error; err != nil {
```

Comment:
cc: @arinda-arif

Reply:
I think project will be needed to get the job run. removing namespace from preload

Reply:
Actually it will break while converting `postgres.job` to `models.jobspec` if we exclude either project or namespace I checked other queries and it follows the same.

Reply:
I see, if that's the case then no need to change it. We can resolve this one.